go-controller/pkg/ovn/ha_master.go

```diff
@@ -3,47 +3,73 @@ package ovn
 import (
 	"context"
 	"os"
+	"strconv"
+	"strings"
 	"time"
 
 	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/config"
 	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/factory"
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/util"
 	"github.com/sirupsen/logrus"
+	kapi "k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
 	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/cache"
 	"k8s.io/client-go/tools/leaderelection"
 	"k8s.io/client-go/tools/leaderelection/resourcelock"
 )
 
 const (
+	invalidIPAddress = "0.0.0.1"
 	haLeaderLockName = "ovn-kubernetes-master"
+	ovnkubeDbEp      = "ovnkube-db"
+	haMasterLeader   = "k8s.ovn.org.ovnkube-master-leader"
```

Comment:
@dcbw @numansiddique this should be `k8s.ovn.org/ovnkube-master-leader`

Reply:
Done